server/calendar/src/main.rs

```diff
@@ -20,14 +20,17 @@ async fn health_status_handler() -> HttpResponse {
         .body(format!("healthy\nsource_code: {github_link}"))
 }
 
+const SECONDS_PER_DAY: u64 = 60 * 60 * 24;
 #[actix_web::main]
 async fn main() -> std::io::Result<()> {
     env_logger::init_from_env(env_logger::Env::default().default_filter_or("info"));
 
-    let last_sync = web::Data::new(Mutex::new(None));
-    let cloned_last_sync = last_sync.clone();
     actix_rt::spawn(async move {
-        scraping::continous_scraping::start_scraping(cloned_last_sync).await;
+        scraping::continous_scraping::start_scraping(
+            Duration::from_secs(SECONDS_PER_DAY),
+            chrono::Duration::days(30 * 4),
```

Comment:
Maybe it's better to also create a constant here (like `EVENTS_TIME_WINDOW_IN_DAYS`, does that make sense?) to make clear what the numbers mean. I understand it that the calendar API provides all events from 2 months before until 2 months after now.

Reply:
I think this would not be the best design choice.
Eventually, I want to have more than these 4 Months, but scraped smarter than front to back. (this is a different PR)
Meanwhile, I am working on a follow-up PR, which adds another scraper (longer `duration` and`scrape_every` with lower request rate)

Reply:
Okay yes if it's just temporary I can understand